src/sentry/workflow_engine/processors/detector.py

```diff
@@ -266,7 +274,7 @@ def process_detectors[T](
                         "detector_resolved",
                         extra=logger_extra,
                     )
-                create_issue_platform_payload(result)
+                create_issue_platform_payload(result, detector)
```

Comment:
for now, can we just pass in the detector.type?

I think longer term we should create a `ProcessDetectorContext` or something similar to what we did in workflow processing to share this stuff between all these methods.